go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22
 require (
 	dagger.io/dagger v0.11.6
 	github.com/dagger/dagger/engine/distconsts v0.11.6
+	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
```

Comment:
:shakes_fist_at_automated_tooling:

Love it, yeah, why not, makes total sense, this block definitely isn't separated for clarity :cry:

Reply:
Yeah that's been annoying. Maybe if we move it below the other block it'll be safe? :thinking: